po/ru.po

```diff
@@ -0,0 +1,355 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Jos Lemmens <jos@jlemmens.nl>
+# This file is distributed under the same license as the ebook-speaker package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ebook-speaker 6.2\n"
+"Report-Msgid-Bugs-To: https://github.com/book-readers/ebook-speaker/issues\n"
+"POT-Creation-Date: 2021-08-21 13:35+0200\n"
+"PO-Revision-Date: 2022-02-15 13:04+0300\n"
+"Last-Translator: Vsevolod Popov\n"
+"Language-Team: Vsevolod Popov <sevapopov13@gmail.com>\n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: src/alsa_ctl.c:38
+msgid "Cannot read /proc/asound/cards"
+msgstr "Невозможно прочесть /proc/asound/cards"
+
+#: src/common.c:33 src/common.c:294 src/common.c:510 src/ebook-speaker.c:2111
+#: src/ebook-speaker.c:2122
+msgid "Please try to play this book with daisy-player"
+msgstr "Пожалуйста, попробуйте воспроизвести эту книгу с помощью Daisy-плеера"
+
+#: src/common.c:773
+#, c-format
+msgid "Cannot read type: %d"
+msgstr "Невозможно прочесть тип: %d"
+
+#: src/common.c:838
+msgid "Go to page number:"
+msgstr "Перейти на страницу: "
+
+#: src/common.c:856 src/ebook-speaker.c:1674 src/ebook-speaker.c:2627
+#: src/ebook-speaker.c:2932
+msgid "Please wait..."
+msgstr "Пожалуйста, подождите..."
+
+#: src/common.c:1096
+msgid "Select a soundcard:"
+msgstr "Выберите звуковую карту:"
+
+#: src/daisy2.02.c:49 src/daisy2.02.c:130 src/daisy2.02.c:236
+#: src/ebook-speaker.c:2514
+#, c-format
+msgid "Cannot read %s"
+msgstr "Невозможно прочесть %s"
+
+#: src/ebook-speaker.c:304
+msgid "Select a Text-To-Speech application"
+msgstr "Выберите синтезатор речи"
+
+#: src/ebook-speaker.c:322
+msgid "Maximum number of TTS's is reached."
+msgstr "Достигнут максимальный набор допустимых синтезаторов речи"
+
+#: src/ebook-speaker.c:682 src/list_dir.c:55 src/list_dir.c:251
+msgid "'h' for help"
+msgstr "'h' для справки"
+
+#: src/ebook-speaker.c:687
+#, c-format
+msgid "%d pages"
+msgstr "%d страниц"
+
+#: src/ebook-speaker.c:692
+#, c-format
+msgid "level: %d of %d"
+msgstr "Уровень: %d из %d"
+
+#: src/ebook-speaker.c:696
+#, c-format
+msgid "total phrases: %d"
+msgstr "Всего фраз: %d"
+
+#: src/ebook-speaker.c:782 src/list_dir.c:165
+msgid "These commands are available in this version:"
+msgstr "Следующие команды доступны в этой версии:"
+
+#: src/ebook-speaker.c:786 src/list_dir.c:169
+msgid "cursor down,2   - move cursor to the next item"
+msgstr "курсор вниз,2   - переместить курсор на следующий элемент"
+
+#: src/ebook-speaker.c:788 src/list_dir.c:171
+msgid "cursor up,8     - move cursor to the previous item"
+msgstr "курсор вверх,8     - переместить курсор на предыдущий элемент"
```

Comment:
This is still incoherent. For instance here we'd get

```
курсор вниз,2   - переместить курсор на следующий элемент
курсор вверх,8     - переместить курсор на предыдущий элемент
```

the dashes are not aligned

Reply:
I try to understand how to fix that, I will describe the steps I do to fix the alignments, I assume they are wrong. For editing I use VSCode. I tried using POEditor but I am not sure how accessible it is with Orca so I chose VSCode. To edit alignments I copy the amount of spaces from msgid to msgstr so the alignment would be the same in these both lines. Is that correct or the other procedure should be done to fix that? How do I understand that alignment is correct? Should I compare it with some other lines? Thank you.

Reply:
No, the question is not about taking the same number of spaces as in english, but about taking the same number of characters before the dash in all russian translations (which doesn't have to be the same as in english). See the help menu in english:

```
cursor right,6  - open this directory or file
cursor left,4   - open previous directory
page-down,3     - view next page
page-up,9       - view previous page
enter,6         - open this directory or file
```

They are aligned because they have the same number of characters before the dash.

Reply:
Ok, thank you for explanation. I am trying to figure out if it's possible to fix the alignment without the help of sited person, because the spaces before dashes are also confusing when counting the symbols so now I try to understand how to fix it on my own, otherwise I will try to use the help of the sited person. But if there are any workarounds about that I will gladly appreciate them. Maybe the editor that I use matters?

Reply:
Since Russian doesn't have wide characters, you can simply count how many right arrow presses you need to go from the starting double-quote to the dash.

Some editors also provide the column position, you can check that it's the same for all dashes of the russian translations.